apps/meteor/tests/end-to-end/apps/video-conferences.ts

```diff
@@ -581,6 +581,33 @@ describe('Apps - Video Conferences', () => {
 								.that.satisfies((msg: string) => msg.includes('Chat History'));
 						});
 				});
+
+				it('should have created the discussion room with hidden subscription flag', async function () {
```

Comment:
not sure if there's a `hidden` subscription flag, maybe we can name it just `should have created a subscription with open = false`